extensions/common/xwalk_external_extension.h

```diff
@@ -39,8 +40,9 @@ class XWalkExternalExtension : public XWalkExtension {
 
   bool Initialize();
 
-  void set_runtime_variables(const base::ValueMap& runtime_variables) {
-    runtime_variables_ = runtime_variables;
+  void set_runtime_variables(const base::DictionaryValue::Storage& runtime_variables) {
+    runtime_variables_.emplace_hint(runtime_variables_.end(), runtime_variables.begin()->first,
```

Comment:
It was a reference to the ValueMap. Second is unique_ptr now and it is not possible to use operator= here.